pypdf/_page.py

```diff
@@ -2340,7 +2340,10 @@ def process_font(f: DictionaryObject) -> None:
             )
         ):
             # the list comprehension ensures there is FontFile
-            emb.add(cast(str, f["/BaseFont"]))
+            try:
+                emb.add(cast(str, f["/BaseFont"]))
+            except KeyError:
+                emb.add("(" + cast(str, f["/Subtype"]) + ")")
```

Comment:
Without looking at the spec myself: Why do we need the brackets here?

Reply:
this not part of the standard. It is just for the user to have an indication of the base name. As there is no name we just tell him that the type of font we rely on.brackets will tell it is not an actual name.

Reply:
Thanks for the explanation.